master/internal/api_experiment.go

```diff
@@ -71,7 +125,13 @@ func (a *apiServer) getExperiment(experimentID int) (*experimentv1.Experiment, e
 	sort.Slice(exp.TrialIds, func(i, j int) bool {
 		return exp.TrialIds[i] < exp.TrialIds[j]
 	})
-	return exp, nil
+
+	expListForm, err := a.enrichExperimentState([]*experimentv1.Experiment{exp})
+	if err != nil {
+		return nil, err
+	}
+
+	return expListForm[0], nil
 }
 
 func (a *apiServer) GetExperiment(
```

Comment:
should GetExperiment also do something new? it'd be weird if the state between `GetExperiment` and `GetExperiments` is different because one is using JobSummary and another is doing the enrichment.

Reply:
`GetExperiment` (the route) goes through `getExperiment` (the helper function) which goes through enrichment (`a.enrichExperimentState([]*experimentv1.Experiment{exp})`)

On review, I've changed the valid states for when to check `GetJobSummary` vs. ending GetExperiment early